addon-test-support/frost-autocomplete.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @typedef {Object} FrostSelectState
+ * @property {Boolean} [disabled=false] - whether or not select is disabled
+ * @property {Boolean} [error=false] - whether or not select has error
+ * @property {Boolean} [focused] - whether or not select is focused
+ * @property {String} [focusedItem] - label of focused item
+ * @property {String} [items] - list of item labels present in dropdown
+ * @property {Boolean} [opened=false] - whether or not select is opened
+ * @property {Number} [tabIndex=0] - tab index of root element
+ * @property {String} [text=''] - text in select for describing what is selected
+ */
+
+import {expect} from 'chai'
+import Ember from 'ember'
+const {$, RSVP, run, typeOf} = Ember // eslint-disable-line
+import {$hook} from 'ember-hook'
+import wait from 'ember-test-helpers/wait'
+
+const assign = Object.assign || Ember.assign || Ember.merge // eslint-disable-line
+
+/* eslint-disable complexity */
+/**
+ * Verify select exists with expected state
+ * @param {jQuery|String} select - name of Ember hook or jQuery instance
+ * @param {FrostSelectState} state - expected select state
```

Comment:
The name of the param is `FrostSelectState`? This is a test helper for `frost-autocomplete`